modin/pandas/test/dataframe/test_join_sort.py

```diff
@@ -348,18 +348,19 @@ def test_sort_index(axis, ascending, na_position):
         for df in [modin_df, pandas_df]:
             df.index = [(i - length / 2) % length for i in range(length)]
 
+    dfs = [modin_df, pandas_df]
     # Add NaNs to sorted index
-    for df in [modin_df, pandas_df]:
-        sort_index = df.axes[axis]
-        df.set_axis(
+    for idx in range(len(dfs)):
+        sort_index = dfs[idx].axes[axis]
+        dfs[idx] = dfs[idx].set_axis(
             [np.nan if i % 2 == 0 else sort_index[i] for i in range(len(sort_index))],
             axis=axis,
-            inplace=True,
+            copy=False,
         )
 
     eval_general(
-        modin_df,
-        pandas_df,
+        dfs[0],
+        dfs[1],
```

Comment:
just revert to original state, this one is even _less_ readable than `*dfs` 😉

Reply:
reverted